pallets/xyk/src/lib.rs

```diff
@@ -269,7 +269,10 @@ const TREASURY_PERCENTAGE: u128 = 5;
 const BUYANDBURN_PERCENTAGE: u128 = 5;
 const FEE_PERCENTAGE: u128 = 30;
 const POOL_FEE_PERCENTAGE: u128 = FEE_PERCENTAGE - TREASURY_PERCENTAGE - BUYANDBURN_PERCENTAGE;
-const TIMEBLOCKRATIO: u32 = 1000;
+// Smallest granularity in which values for liquidity minting rewards change
+const TIMEBLOCKRATIO: u32 = 10000;
```

Comment:
maybe that value should be passed as one of xyk config parameter

sth lik https://github.com/mangata-finance/mangata-node/blob/b90f22213d6cc120078a5007dc6423549b8a6104/pallets/xyk/src/lib.rs#L303